.github/workflows/publish_unit_tests.yml

```diff
@@ -0,0 +1,62 @@
+name: Unit Test Results
+
+on:
+  workflow_run:
+    workflows: ["core_build_publish"]
+    types:
+      - completed
+
+jobs:
+  unit-test-results:
+    name: Unit Test Results
+    runs-on: ubuntu-latest
+    if: >
+      github.event.workflow_run.conclusion != 'skipped' && (
+         github.event.sender.login == 'dependabot[bot]' ||
+         github.event.workflow_run.head_repository.full_name != github.repository
+      )
+
+    steps:
+      - name: Download Artifacts
+        uses: actions/github-script@v3.1.0
+        with:
+          script: |
+            var fs = require('fs');
+            var path = require('path');
+            var artifacts_path = path.join('${{github.workspace}}', 'artifacts')
```

Comment:
`var` → `const`?

Reply:
Since there is no frontend here, I hereby reject this request - this is nowhere being `const`ant (for the real meaning of constant). :P

Reply:
So, `var` →  `let` maybe?

Reply:
Doesn't change anything here IMO.